pkg/api/install/install.go

```diff
@@ -128,3 +130,15 @@ func interfacesFor(version unversioned.GroupVersion) (*meta.VersionInterfaces, e
 		return nil, fmt.Errorf("unsupported storage version: %s (valid: %v)", version, g.GroupVersions)
 	}
 }
+
+func addVersionsToScheme(externalVersions ...unversioned.GroupVersion) {
+	// add the internal version to Scheme
+	api.Register()
+	// add the enabled external versions to Scheme
+	for _, v := range externalVersions {
+		switch v {
+		case v1.SchemeGroupVersion:
+			v1.Register()
```

Comment:
Eventually this should be gated on a `register.IsEnabled(GroupVersion)` call, right?

```go
switch v {
case v1.SchemeGroupVersion && register.IsEnabled(v):
    v1.Register()
```